### DISCUSSION TITLE
Where is installed Portainer - Docker LXC

### DISCUSSION THREAD

**Discussion by Baptiste-mrch** [tteck/Proxmox#2110] [Help Needed]
Hi,
I want to remove Portainer because of an production environment. Where it is installed ?
Thanks!

**Comment by tteck** [comment]
In a Docker container. Google "how to remove a Docker container"

https://github.com/tteck/Proxmox/blob/4b976b31dcaf7b992a266aa0eb143e193c244272/install/docker-install.sh#L50-L58

**Reply by Baptiste-mrch** [reply]
Alright I just thought that it was inside some sort of docker compose but no, my bad.
Thanks :)
